ui/src/components/Configuration/ConfigurationToast.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div class="config-toast" :class="getToastClass()">
+    <div class="message">
+      <div class="icon">
+        <FeatherIcon
+          class="icon-inner"
+          :icon="toastMessage?.hasError ? editIcon : successIcon"
+          :class="getIconClass()"
+        />
+      </div>
+      <div class="basic">{{ toastMessage?.basic }}</div>
+      <div class="detail">{{ toastMessage?.detail }}</div>
+      <div class="close">
+        <FeatherButton icon="Cancel" text @click="closeToast">
+          <FeatherIcon class="close-icon" :icon="cancelIcon" />
+        </FeatherButton>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+
+import { computed, reactive, watch } from 'vue'
+import { useStore } from 'vuex'
+
+import { FeatherIcon } from '@featherds/icon'
+import { FeatherButton } from '@featherds/button'
+
+import Edit from '@featherds/icon/action/Edit'
+import CheckCircle from '@featherds/icon/action/CheckCircle'
+import Cancel from '@featherds/icon/navigation/Cancel'
+
+const store = useStore()
+
+/**
+ * Local State
+ */
+const toastMessage = computed(() => store?.state?.configuration?.toast)
```

Comment:
This toast is pretty cool. 
Doesn't have to be done here, but at some point we might want to untie this from config and bring it into the common hooks section to be used elsewhere in the app.